ui/app/AppLayouts/Profile/stores/ProfileSectionStore.qml

```diff
@@ -101,9 +104,6 @@ QtObject {
             append({subsection: Constants.settingsSubsection.devicesSettings,
                        text: qsTr("Devices settings"),
                        icon: "mobile"})
-            append({subsection: Constants.settingsSubsection.browserSettings,
```

Comment:
@endulab as ticket says... browser should be under the "Apps" section but when we enable it in the "Advanced" section, and "Apps" section should be visible only if there is at least one item in it and so far only "Browser" belong to that section.

Reply:
Indeed this setting is not showing and the browser setting are gone

https://user-images.githubusercontent.com/176720/156637097-8d10f0db-06c0-41b5-968e-6d1db2e7517e.mov

Reply:
Thanks @saledjenic for catching this. I fixed the issue.
I also saw that the Wallet settings are visible only when "Multi network" is on. 
It does not depend on Wallet toggle. Is it the correct behaviour @iurimatias ?
Maybe we can improve it within this ticket ?